packages/azure/src/application/AzureAccount.ts

```diff
@@ -88,7 +93,7 @@ export default class AzureAccount extends CloudProviderAccount {
           return await this.getDataForSubscription(
             startDate,
             endDate,
-            subscription.subscriptionId,
+            subscription.subscriptionId || '',
```

Comment:
It might be good to reference the variable that you created here
```suggestion
            subscription.subscriptionId || defaultSubscriptionId,
```